xbit/config.template.js

```diff
@@ -6,5 +6,8 @@ exports = module.exports = {
     ssl: false,
     port: 8080,
     dns: "www.xbit.ga",
-    address: "127.0.0.1"
```

Comment:
do not add anything real in the template file. It must be copied to config.js, and in config.js we save the real values (like AWS key, etc.)